apps/lead/serializers.py

```diff
@@ -717,3 +720,42 @@ def validate(self, data):
         data['project'] = self.project
         data['user'] = self.current_user
         return data
+
+
+class DeduplicationCallbackSerializer(serializers.Serializer):
+    lead_id = serializers.IntegerField()
```

Comment:
Shouldn't this be char?

Reply:
Nice catch. Yes, we need to use `client_id` (char) instead of lead_id. As we use callback without authentication, we use client_id to validate the request. client_id is generated using lead_id and a signature using Django-secret. You can check out ExtractCallbackSerializer for more info.

Reply:
Done!